test/app.hooks.test.js

```diff
@@ -255,6 +298,8 @@ describe('Sanitization Hook', () => {
 
 		expect(result.name).to.equal('');
 	});
+
+	
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5768742)